src/unistd.h

```diff
@@ -0,0 +1,49 @@
+#ifndef _UNISTD_H
```

Comment:
Looks like this is only needed for `isatty` in `wasm-flex-lexer.c`, is that right? If so, just remove this file and do the `#define` there.